packages/create-astro/test/create-astro.test.js

```diff
@@ -2,119 +2,137 @@ import fs from 'fs';
 import path from 'path';
 import { fileURLToPath } from 'url';
 import http from 'http';
-import { suite } from 'uvu';
```

Comment:
`uvu` was removed for this test because it was just swallowing error after error, making it impossible to debug in CI